src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -157,6 +158,16 @@ public ObservableList<Person> getFilteredPersonList() {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void addGroup(Group group) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
Im guessing these are just dummy methods that will be replaced in the near future.

Reply:
I think this is the stub. All its methods throws the AssertionError 😆

Reply:
This ModelStub is only for testing of the AddCommand which doesn't require this method. Future unit tests for this command will include more comprehensive testing